package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -560,10 +560,16 @@ def _read_next_timestep(self):
 
         triclinic = len(f.readline().split()) == 9  # ITEM BOX BOUNDS
         if triclinic:
-            xlo, xhi, xy = map(float, f.readline().split())
-            ylo, yhi, xz = map(float, f.readline().split())
+            xlo_bound, xhi_bound, xy = map(float, f.readline().split())
+            ylo_bound, yhi_bound, xz = map(float, f.readline().split())
             zlo, zhi, yz = map(float, f.readline().split())
 
+            # converts orthogonal bounding box to the conventional format, see https://docs.lammps.org/Howto_triclinic.html
+            xlo = xlo_bound - min(0.0, xy, xz, xy + xz)
+            xhi = xhi_bound - max(0.0, xy, xz, xy + xz)
+            ylo = ylo_bound - min(0.0, yz)
+            yhi = yhi_bound - max(0.0, yz)
```

Comment:
Agrees with https://docs.lammps.org/Howto_triclinic.html  — I am glad that LAMMPS is documented so well. ✅